packages/owl-bot/src/core.ts

```diff
@@ -205,7 +205,9 @@ async function waitForBuild(
   id: string,
   client: CloudBuildClient
 ): Promise<google.devtools.cloudbuild.v1.IBuild> {
-  for (let i = 0; i < 60; i++) {
+  // This loop is set to equal a total of 3 hours, which should
+  // match the timeout in cloud-build/update-pr.yaml's timeout
+  for (let i = 0; i < 540; i++) {
```

Comment:
nit: I would pull the 540 and 20000 out into variables, so that the duration of the wait speaks for itself:

const THREE_HOURS = 3 * 60 * 60 * 1000;
const delay = 20000;
const iterations = parseInt(threeHours / 20000);